File: C1.UWP.FlexChart/CS/StockChart/StockChart/ViewModel/ExportCommand.cs
using C1.Chart;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.ComponentModel;
using Windows.UI.Xaml;
using System.IO;
using C1.Xaml.Chart;
using Windows.Storage.Pickers;
using Windows.Storage;

namespace StockChart.ViewModel
{

    public class ExportCommand : DependencyObject, System.Windows.Input.ICommand
    {
        public event EventHandler CanExecuteChanged;

        public void RaiseCanExecuteChanged()
        {
            if (this.CanExecuteChanged != null)
            {
                this.CanExecuteChanged(this, new EventArgs());
            }
        }

        ChartViewModel _viewModel;
        public ExportCommand(ChartViewModel viewModel)
        {
            _viewModel = viewModel;
        }

        public bool CanExecute(object parameter)
        {
            return parameter != null;
        }

        public async void Execute(object parameter)
        {
            string type = parameter.ToString();


            string fmt = string.Empty;
            C1.Xaml.Chart.ImageFormat format = C1.Xaml.Chart.ImageFormat.Png;
            switch (type)
            {
                case "PNG":
                    fmt = "png";
                    format = C1.Xaml.Chart.ImageFormat.Png;
                    break;
                case "JPG":
                    fmt = "jpg";
                    format = C1.Xaml.Chart.ImageFormat.Jpeg;
                    break;
                case "SVG":
                    fmt = "svg";
                    format = C1.Xaml.Chart.ImageFormat.Svg;
                    break;
                default:
                    fmt = "png";
                    format = C1.Xaml.Chart.ImageFormat.Png;
                    break;
            }
            var picker = new FileSavePicker();

            picker.FileTypeChoices.Add(fmt.ToString().ToUpper(), new List<string>() { "." + fmt.ToString().ToLower() });

            StorageFile file = await picker.PickSaveFileAsync();
            if (file != null)
            {
                Stream stream = new MemoryStream();
                var extension = file.FileType.Substring(1, file.FileType.Length - 1);
                _viewModel.MainChart.SaveImage(stream, format);
                using (Stream saveStream = await file.OpenStreamForWriteAsync())
                {
                    stream.CopyTo(saveStream);
                    stream.Seek(0, SeekOrigin.Begin);
                    saveStream.Seek(0, SeekOrigin.Begin);
                    saveStream.Flush();
                    saveStream.Dispose();
                }
            }


        }

    }
}

File: C1.UWP.FlexChart/VB/StockChart/StockChart/ViewModel/ExportCommand.vb
Imports Windows.Storage
Imports Windows.Storage.Pickers

Public Class ExportCommand
    Inherits DependencyObject
    Implements ICommand
    Public Event CanExecuteChanged As EventHandler Implements ICommand.CanExecuteChanged

    Public Sub RaiseCanExecuteChanged()
        RaiseEvent CanExecuteChanged(Me, New EventArgs())
    End Sub

    Private _viewModel As ChartViewModel
    Public Sub New(viewModel As ChartViewModel)
        _viewModel = viewModel
    End Sub

    Public Function CanExecute(parameter As Object) As Boolean Implements ICommand.CanExecute
        Return parameter IsNot Nothing
    End Function

    Public Async Sub Execute(parameter As Object) Implements ICommand.Execute
        Dim type As String = parameter.ToString()


        Dim fmt As String = String.Empty
        Dim format As C1.Xaml.Chart.ImageFormat = C1.Xaml.Chart.ImageFormat.Png
        Select Case type
            Case "PNG"
                fmt = "png"
                format = C1.Xaml.Chart.ImageFormat.Png
                Exit Select
            Case "JPG"
                fmt = "jpg"
                format = C1.Xaml.Chart.ImageFormat.Jpeg
                Exit Select
            Case "SVG"
                fmt = "svg"
                format = C1.Xaml.Chart.ImageFormat.Svg
                Exit Select
            Case Else
                fmt = "png"
                format = C1.Xaml.Chart.ImageFormat.Png
                Exit Select
        End Select
        Dim picker = New FileSavePicker()

        picker.FileTypeChoices.Add(fmt.ToString().ToUpper(), New List(Of String)() From {
        "." + fmt.ToString().ToLower()
    })

        Dim file As StorageFile = Await picker.PickSaveFileAsync()
        If file IsNot Nothing Then
            Dim stream As Stream = New MemoryStream()
            Dim extension = file.FileType.Substring(1, file.FileType.Length - 1)
            _viewModel.MainChart.SaveImage(stream, format)
            Using saveStream As Stream = Await file.OpenStreamForWriteAsync()
                stream.CopyTo(saveStream)
                stream.Seek(0, SeekOrigin.Begin)
                saveStream.Seek(0, SeekOrigin.Begin)
                saveStream.Flush()
                saveStream.Dispose()
            End Using
        End If


    End Sub

End Class